pkg/storage/blockdev_test.go

```diff
@@ -0,0 +1,24 @@
+package storage
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestFindMountPointNoError checks that there is no
+// error in parsong the test output.
+func TestFindMountPointNoError(t *testing.T) {
```

Comment:
this test is actually not necessary, as it is covered by the test below.

It would actually be useful to test the case where the device is not in the mounts file

Reply:
Done